storage/flux/table.gen.go

```diff
@@ -86,6 +86,10 @@ func (t *floatTable) Do(f func(flux.ColReader) error) error {
 
 func (t *floatTable) advance() bool {
 	a := t.cur.Next()
+	if t.cur.Err() != nil {
+		t.err = t.cur.Err()
+		return false
+	}
 	l := a.Len()
 	if l == 0 {
 		return false
```

Comment:
Other uses of cursor:

```
servePromRead calls cur.Next

floatLimitArrayCursor etc in storage/reads/array_cursor.gen.go

Possibly some of the uses of cursor in storage/merge? though I think this is pretty good, we at least check for errors before closing the cursor

streamFloatArrayPoints etc in response_writer.gen.go


```

Reply:
@lesam - are you asking for changes beyond the infrequent check for errors?

Reply:
AFAICT `servePromRead` doesn't check for errors at all - I'm saying this PR doesn't fully address error checks on cursors

Reply:
Which to be clear is fine, but we should at least have a follow-up issue to check on errors outside this file